sycl/include/sycl/reduction.hpp

```diff
@@ -823,6 +823,7 @@ void reduSaveFinalResultToUserMem(handler &CGH, Reduction &Redu) {
 namespace reduction {
 template <typename KernelName, strategy S, class... Ts> struct MainKrn;
 template <typename KernelName, strategy S, class... Ts> struct AuxKrn;
+template <typename KernelName, aspect... A> struct AspectDepKrn;
```

Comment:
Discussed offline - more investigation is needed. Both Main/AuxKrn accept strategy so there shouldn't be duplicate kernel names.